coral-common/src/main/java/com/linkedin/coral/common/transformers/SqlCallTransformer.java

```diff
@@ -49,11 +50,19 @@ public SqlCallTransformer(SqlValidator sqlValidator) {
   protected abstract SqlCall transform(SqlCall sqlCall);
 
   /**
-   * Public entry of the transformer, it returns the result of transformed SqlCall if `predicate(SqlCall)` returns true,
+   * Public entry of the transformer, it returns the result of transformed SqlCall if `condition(SqlCall)` returns true,
    * otherwise returns the input SqlCall without any transformation
    */
   public SqlCall apply(SqlCall sqlCall) {
     if (sqlCall instanceof SqlSelect) {
+      // Updates selectList to correctly handle t.* type SqlSelect sqlNodes for accurate data type derivation
```

Comment:
Could you clarify `t.*` here with a SQL example, and elaborate what kind of SqlSelect will be stored for that?